src/main/java/org/datadog/jmxfetch/JmxAttribute.java

```diff
@@ -191,16 +194,31 @@ private List<String> getBeanParametersList(
         return beanTags;
     }
 
+    private String unquote(String beanParameter) {
```

Comment:
Lets add a brief comment describing the try-catch here. Something to help future readers understand that if the `beanParameter` doesn't contain quotes, its expected to hit the catch and return the original.